oracle/project/fuel-toolchain.toml

```diff
@@ -1,6 +1,6 @@
 [toolchain]
-channel = "latest-2023-04-12"
+channel = "latest-2023-07-07"
 
 [components]
-forc = "0.37.0"
-fuel-core = "0.17.9"
+forc = "0.42.1"
+fuel-core = "0.18.2"
```

Comment:
According to [this release](https://github.com/FuelLabs/fuelup/blob/gh-pages/channels/latest/channel-fuel-latest-2023-07-07.toml#L81) we are 1 patch ahead. I'm unsure of the reason behind your choice of 18.2 but I'll leave it to you to bump or leave it at this

Reply:
I chose the same version of `fuel-core` that `forc 0.42.1` uses